templates/scenarios/ts/link-unfurling/README.md

```diff
@@ -0,0 +1,232 @@
+| :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| <ul><li>Open Teams Toolkit, and sign into Azure by clicking the `Sign in to Azure` under the `ACCOUNTS` section from sidebar.</li> <li>After you signed in, select a subscription under your account.</li><li>Open the Teams Toolkit and click `Provision` from DEPLOYMENT section or open the command palette and select: `Teams: Provision`.</li><li>Open the Teams Toolkit and click `Deploy` or open the command palette and select: `Teams: Deploy`.</li></ul> | <ul> <li>Run command `teamsfx account login azure`.</li> <li>Run command `teamsfx provision --env dev`.</li> <li>Run command: `teamsfx deploy --env dev`. </li></ul> |
+
+> Note: Provisioning and deployment may incur charges to your Azure Subscription.
+
+## Preview
+
+Once the provisioning and deployment steps are finished, you can preview your app:
+
+- From Visual Studio Code
+
+  1. Open the `Run and Debug Activity Panel`.
+  1. Select `Launch Remote in Teams` or `Launch Remote in Outlook` from the launch configuration drop-down.
+  1. Press the Play (green arrow) button to launch your app - now running remotely from Azure.
+
+- From TeamsFx CLI: execute `teamsfx preview --env dev    --m365-host <m365-host>` in your project directory to launch your application, where options for m365-host are `teams` or `outlook`.
+## How to add link unfurling cache in Teams
+This template removes cache by default to provide convenience for debug. To add cache, remove following JSON part from adaptive card in `linkUnfurlingBot.ts`:
+```ts
+suggestedActions: {
+          actions: [
+            {
+              title: "default",
+              type: "setCachePolicy",
+              value: '{"type":"no-cache"}',
+            },
+          ],
+        }
+```
+After removing this, the link unfurling result will be cached in Teams for 30 minutes. 
+
+Please refer to [this document](https://learn.microsoft.com/en-us/microsoftteams/platform/messaging-extensions/how-to/link-unfurling?tabs=desktop%2Cjson%2Cadvantages#remove-link-unfurling-cache) for more details.
+
```

Comment:
Can we reformat the README? Something similar with command bot. We need to include:
* An overview of this template.
* Get Started that includes Prerequisites, and how to run this app in Teams/Outlook.
* What's included in the template, a folder structure walkthrough.
* Extend this template (Customization Scenario)
  * How to use Zero Install
  * How to add stage view / task module / adaptive card actions
  * How to customize adaptive cards.
  * How to handle regular expressions with unfurled links? 
  * etc...

Reply:
updated.